### PR TITLE
Organization People Revamp design 

### LINKED ISSUE #1550: Admin Redesign: Redesiging the People Management Screen
**Is your feature request related to a problem? Please describe.**
we want consistent design for  people management

**Describe the solution you'd like**
redesign the people management as of the designs in the figma file

**Additional context**
Screenshot 2024-02-06 at 10 09 00 PM

link to the figma file: https://www.figma.com/file/WksD4uIa5sQJMIk3pR4l0c/Talawa-Design-System-(Official)?type=design&node-id=307-212&mode=design&t=jw1Q8PosWMSmrFRM-0

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by duplixx**
**What kind of change does this PR introduce?**

Design Revamp

**Issue Number:**

Fixes #1550 

**Did you add tests for your changes?**

**Snapshots/Videos:**
image

https://github.com/PalisadoesFoundation/talawa-admin/assets/90516956/bb84254d-fe29-41c6-9f6e-b600f13eb894

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Review by rishav-jha-mech — CHANGES_REQUESTED** [review]
@duplixx why change talawa-admin-docs ?, revert those changes

File: src/screens/OrganizationPeople/OrganizationPeople.tsx [hunk 1]
```diff
@@ -1,30 +1,51 @@
...
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell, { tableCellClasses } from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import { styled } from '@mui/material/styles';
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
```
Comment (rishav-jha-mech) [inline src/screens/OrganizationPeople/OrganizationPeople.tsx:28]: Remove this comment

File: src/screens/OrganizationPeople/OrganizationPeople.tsx [hunk 2]
```diff
@@ -197,12 +230,10 @@ function organizationPeople(): JSX.Element {
...
                         setState(2);
                       }}
-                    />
-                    <label htmlFor="userslist">{t('users')}</label>
-                  </div>
-                  <div>
-                    <Form.Check
-                      type="radio"
+                    >
+                      <label htmlFor="userslist">{t('users')}</label>
```
Comment (rishav-jha-mech) [inline src/screens/OrganizationPeople/OrganizationPeople.tsx:234]: Use Form.Label form Bootstrap instead

File: src/screens/OrganizationPeople/OrganizationPeople.tsx [hunk 3]
```diff
@@ -227,187 +256,257 @@ function organizationPeople(): JSX.Element {
...
-                                  memberImage={datas.image}
-                                  joinDate={dayjs(datas.createdAt).format(
-                                    'DD/MM/YYYY'
+          </div>
+        </Row>
+        <Col sm={9}>
+          <div className={styles.mainpageright}>
+            {memberLoading || usersLoading || adminLoading ? (
+              <>
+                <div className={styles.loader}></div>
```
Comment (rishav-jha-mech) [inline src/screens/OrganizationPeople/OrganizationPeople.tsx:285]: User Loader Component from `src/components/Loader/Loader.tsx`

File: src/screens/OrganizationPeople/OrganizationPeople.tsx (hunk 1 above)
Comment (duplixx) [reply src/screens/OrganizationPeople/OrganizationPeople.tsx:28]: hey @rishav-jha-mech when I am removing I am getting this error ,I am using the functions as wrapper so it needs to be like this,  how should i fix it 
image

Comment (rishav-jha-mech) [reply src/screens/OrganizationPeople/OrganizationPeople.tsx:28]: @duplixx so have you tried using camelCase or UPPER_CASE variable name ?

Comment (duplixx) [reply src/screens/OrganizationPeople/OrganizationPeople.tsx:29]: @rishav-jha-mech yeah i did that

**Comment by palisadoes** [comment]
Merge with the latest upstream after reverting documentation.

**Comment by duplixx** [comment]
@palisadoes all conflicts are resolved ,is fetching upstream needed now?

**Comment by palisadoes** [comment]
1. No. That should be OK
2. Please fix the tests

**Comment by aashimawadhwa** [comment]
@duplixx  any updates?

**Comment by duplixx** [comment]
I'll be working on it super soon currently having end term exams till 17th

**Comment by palisadoes** [comment]
@aashimawadhwa is working on a design to be used for this. It will be shared soon

**Comment by palisadoes** [comment]
_**NOTE**_ Read very carefully

1. We just merged this PR which upgraded the prettier package. 
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/1599
1. It reformatted over 150 files. 

This will put your PR at risk of extensive merge conflicts. Do the following **IN THIS ORDER**:

1. upgrade your `prettier` in your local branch to the same version
1. run `prettier` on your local branch
1. update your local branch with the latest upstream
    
This will help to reduce the number of **existing** and **future** merge conflicts for your PR.

File: src/screens/OrganizationPeople/OrganizationPeople.module.css [hunk 4]
```diff
@@ -208,17 +199,94 @@
...
+  border-radius: 20px !important;
+  padding: 20px;
+  margin-top: 20px;
+}
+
+.tableHead :nth-first-child() {
+  border-top-left-radius: 20px;
+}
+
+.mainpageright > hr {
```
Comment (rishav-jha-mech) [inline src/screens/OrganizationPeople/OrganizationPeople.module.css:284]: Use a single line to add all the margins

Comment (duplixx) [reply src/screens/OrganizationPeople/OrganizationPeople.module.css:284]: yeah sure

**Comment by duplixx** [comment]
@aashimawadhwa let me know when i can get new design

**Comment by aashimawadhwa** [comment]
@duplixx  Here are some suggestion which you should work upon :

1.  Add the actions button 
2.  render profile icon (light green icon as per the designs) when no image is displayed.
3.  make sure the user details is accessible when. clicked on the name (attached a video for the reference).

https://github.com/PalisadoesFoundation/talawa-admin/assets/73706697/9f3c732b-1447-4767-aa1f-c80961d7dc1b

**Comment by rishav-jha-mech** [comment]
@duplixx also. the new design does not includes infinite scroll, please remove the pagination section from this screen.

You can refer to Organisations screen to get a reference on how to implement infinite scrolling.

**Comment by duplixx** [comment]
> @duplixx also. the new design does not includes infinite scroll, please remove the pagination section from this screen.
> 
> You can refer to Organisations screen to get a reference on how to implement infinite scrolling.

@rishav-jha-mech, just for clarification: Do I need to stick with this design and add an Action button on the column, while removing pagination? Or, as you suggested, should I follow a cards approach to display data in the Organization People section?
image

**Comment by duplixx** [comment]
@aashimawadhwa @rishav-jha-mech Here's the updated design with action buttons and removed pagination. The table is scrollable on the x-axis.
image
image

File: src/screens/OrganizationPeople/OrganizationPeople.test.tsx [hunk 5]
```diff
@@ -941,10 +825,10 @@ describe('Organization People Page', () => {
       </MockedProvider>,
     );
     await wait();
-    userEvent.click(screen.getByLabelText(/Users/i));
-    await wait();
-    expect(screen.getByLabelText(/Users/i)).toBeChecked();
-    await wait();
+    // userEvent.click(screen.getByLabelText(/Users/i));
```
Comment (rishav-jha-mech) [inline src/screens/OrganizationPeople/OrganizationPeople.test.tsx:828]: @duplixx why have you commented the tests ?

Comment (duplixx) [reply src/screens/OrganizationPeople/OrganizationPeople.test.tsx:828]: these tests are not used as by requirements we dont pagination, radio buttons and type of member label

**Review by palisadoes — APPROVED** [review]

**Pull request merged**